package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "@playwright/test": "^1.23.3",
+    "babel-preset-env": "^1.7.0",
+    "lodash": "^4.17.21",
+    "parcel-bundler": "^1.12.5"
+  }
+}
```

Comment:
can you please merge this file and the package-lock file with the ones in `src/frontend`?

Reply:
Fixed